spi/src/main/java/com/lightbend/lagom/internal/spi/ServiceDiscovery.java

```diff
@@ -4,10 +4,18 @@
 package com.lightbend.lagom.internal.spi;
 
 import java.util.List;
+import java.util.Optional;
 
 /**
  * This class can be implemented by a Play ApplicationLoader to allow tooling to detect services and their ACLs.
  */
 public interface ServiceDiscovery {
+    /**
+     * @deprecated support for multiple locatable ServiceDescriptors per Lagom service was
+     *             removed in 1.3.1. Use {@link ServiceDiscovery@discoverService} instead
```

Comment:
The syntax error is fixed in an ulterior commit. I'll review other comments because it's not the first time I make that typo.

(performs some `bash` spells)
```
find . | grep -v target | grep "\.java$"| xargs grep "link [^}]*@[^}]*}"
```

OK, I couldn't find any other similar typo on the tip of this PR.